test/errtest.c

```diff
@@ -74,7 +74,10 @@ static int test_print_error_format(void)
         goto err;
     }
 
-# ifndef OPENSSL_NO_ERR
+# if defined(OPENSSL_NO_AUTOERRINIT)
```

Comment:
Should this `if defined` be actually inside the `if !defined OPENSSL_NO_ERR` or does no-err imply the OPENSSL_NO_AUTOERRINIT being undefined?

Reply:
There are independent I think which means you are correct about the ordering.

Fix submitted.